doc/pravega-options.md

```diff
@@ -14,3 +14,17 @@ spec:
       metrics.statsdPort: "8125"
 ...
 ```
+### Pravega JVM Options
```

Comment:
Lets explain may be in one line how these are different from "options" under PravegaSpec.

Reply:
Yes this looks good.